api/src/test/java/org/openmrs/util/databasechange/DatabaseUpgradeTestUtil.java

```diff
@@ -245,9 +246,6 @@ public SessionFactory buildSessionFactory() {
 		config.setProperty(Environment.USE_SECOND_LEVEL_CACHE, "false");
 		config.setProperty(Environment.USE_QUERY_CACHE, "false");
 		
-		//Let's validate HBMs against the actual schema
-		config.setProperty(Environment.HBM2DDL_AUTO, "validate");
```

Comment:
Did you intentionally remove this?

Reply:
Yes. Validation fails with a datatype that is not used: 

org.hibernate.HibernateException: Wrong column type in OPENMRS.PUBLIC.CONCEPT_ATTRIBUTE for column value_reference. Found: clob, expected: varchar{{ }}

I could not find any value_reference of type clob. Not sure what is going on here.

Reply:
Here I need some advice from the community as value_reference has not the datatype clob.

Reply:
Did you at last get time to find advice on this?

Reply:
Resolved by `ValidateHibernateMappingsDatabaseIT`